source/layouts/dicustom.erb

```diff
@@ -10,6 +10,29 @@
     <%= stylesheet_link_tag :manifest %>
 
     <link rel="canonical" href="<%= meta_tags.canonical_url %>">
+    <% if config[:tech_docs][:show_govuk_logo] %>
+      <%
+        path_prefix =
+          if !development? && config[:tech_docs][:use_service_link_for_favicon]
```

Comment:
slightly confused about the `use_service_link_for-favicon` element here, what's that for?

Seems to be that if we're not in dev (so building via the CLI, AND you have this then... go adapt the service link?
What's the story here about the conditionality?